codecov/urls.py

```diff
@@ -25,4 +25,5 @@
     path("staticanalysis/", include("staticanalysis.urls")),
     path("labels/", include("labelanalysis.urls")),
     re_path(r"^redirect_app", views.redirect_app),
+    path("monitoring/", include("django_prometheus.urls")),
```

Comment:
Does this become publicly available?  Or is `django_prometheus` doing something internal such that this URL is only available on port 8001?

Reply:
good question! talking about those details with @rfox-codecov

as-is it would be public, and if we want to avoid leaking the metrics we would want to add a null-route rule to the URL mapping we do in kubernetes in production. i think prometheus requests the host directly, bypassing the null-route

if we don't like that solution, a couple other options which would require changes in this PR:
- create HTTP Basic Auth credentials for prometheus to supply and wrap this view to perform basic auth
- because prometheus requests the host directly, we could branch on `request.META["Host"]`. if it's an internal request we serve it, else 404

Reply:
i think we're going for the first which means no code change, but i'll add a comment disclaiming that without some higher-level configuration this route will be public